app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PostsController < ApplicationController
+  skip_before_action :verify_authenticity_token
   def index
     @user = User.find(params[:user_id])
```

Comment:
- [ ] Kindly create a `LikesController` to enable a user to like a post. Take a look at the code snippet below for inspiration:
```
  class LikesController < ApplicationController
    ...

    def create
      @post = Post.find(params[:post_id])
      @like = @current_user.likes.new(post_id: @post.id)

      respond_to do |format|
        if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
      end
    end
	  ...

  end
```
- [ ] Also ensure that you add routes for the likes.